.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: ruby
 
 rvm:
-  - 2.3.0
-  - 2.3.3
-  - 2.4.0
+  - 2.2.10
```

Comment:
I don't want to support Ruby 2.2 as it is no longer supported: https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/